src/main/scala/org/apache/spark/metrics/source/ProcessExecutorMetrics.scala

```diff
@@ -2,18 +2,31 @@ package org.apache.spark.metrics.source
 
 import com.codahale.metrics.{Gauge, MetricRegistry}
 import com.nec.ve.VeProcessMetrics
-
 import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
 
 final class ProcessExecutorMetrics() extends VeProcessMetrics with Source {
   private val allocations: scala.collection.mutable.Map[Long, Long] = mutable.Map.empty
+  private val veCalls: ArrayBuffer[Long] =  new ArrayBuffer[Long]()
+  private var totalTransferTime: Long = 0L
+  private var totalSerializationTime: Long = 0L
 
   override def registerAllocation(amount: Long, position: Long): Unit =
     allocations.put(position, amount)
 
+  override def increaseSerializationTime(increaseBy: Long): Unit = {
+    totalSerializationTime += increaseBy
```

Comment:
can emit 2 metrics from here: the total, and also per each serialization, use a `Histogram` as a metric.

Reply:
👍